<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="LabelAmplitude">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Amplitude&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderAmplitude">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber">
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="LabelFrequencia">
            <property name="text">
             <string>Frequencia</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderFrequencia">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_2">
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="LabelVelocidade">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Velocidade&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderVelocidade">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_3">
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="30,70">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCopia">
            <property name="text">
             <string>&gt;&gt;&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Plotter" name="widget_2" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumberX"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumberY"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonQuit">
      <property name="text">
       <string>Morre, diabo!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>recursos/icone.png</normaloff>recursos/icone.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <slot>setAmplitude(int)</slot>
    <slot>setFrequencia(int)</slot>
    <slot>setVelocidade(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>mataTudo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCopia</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>copiaTexto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAmplitude</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderFrequencia</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderVelocidade</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAmplitude</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget_2</receiver>
   <slot>setAmplitude(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderFrequencia</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget_2</receiver>
   <slot>setFrequencia(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderVelocidade</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget_2</receiver>
   <slot>setVelocidade(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>copiaTexto()</slot>
  <slot>mataTudo()</slot>
 </slots>
</ui>
